plugins/ci/version.txt

```diff
@@ -1 +1 @@
-5.2.1
+5.2.2
```

Comment:
I don't see any changes here--did they get rebased away?

Reply:
Should be fixed now